dysnix/bsc/templates/statefulset.yaml

```diff
@@ -98,7 +98,7 @@ spec:
           {{- if kindIs "float64" (get .Values.bsc "history.transactions") }}
           - --history.transactions={{ int (get .Values.bsc "history.transactions") }}
           {{- end }}
-          {{- if .Values.bsc.preimages }}
+          {{- if get .Values.bsc "cache.preimages" }}
```

Comment:
Is there any chance we'll need other options inside `cache`/`history` sections? It may be better to use yaml maps instead of dot-containing keys then, to keep checks to look the same, w/o using `get`

Reply:
You're right, this case may be in future.

Reply:
@voron The only culprit is that `cache` option is already here and it is equivalent to `--cache` CLI argument. So if going with yaml maps we need to rename it too. Do you have any suggestions regarding this?

Reply:
Hm, it seems, a lot more args are the same, f.e  `--http` and `--ws`.
Possible option may look like following for non-boolean values:
```yaml
cache:
  value: 8192
  preimages: true
```
and for boolean values:
```yaml
ws:
  enabled: true
  port: 8546
```  
It's unlikely to get some cli arg with name `value` in the future.
Geth CLI options evolved over time, thus there may be some inconsistence too. 

It may be a good option to bump chart version to 0.7 when we hit incompatible values changes

Reply:
I hope we can do some extra chart refactoring when some time has passed since BSC 1.2.x is deprecated completely.